### PR TITLE
Feat(Contacts): Profile showcase contacts

### LINKED ISSUE #11081: Profile showcase backend
### Description

The UI for this feature is ready but we need to implement the backend part

### PR DESCRIPTION
**Pull request by MishkaRogachev**
Waits https://github.com/status-im/status-desktop/pull/12510
Close #11081

### What does the PR do

This PR adds the ability to observe profile showcase for another contacts. A few leftovers, i'm currently working on:
- [x] Live update on change
- [x] Community description and members count
- [x] Loading state for communities
- [x] Storybook

### Affected areas

Profile, Contacts

### Screenshot of functionality (including design for comparison)

https://github.com/status-im/status-desktop/assets/2522130/f1137f6d-67f8-4f0a-9f4d-b7f588472d07

### CODE REVIEW

File: ui/imports/shared/views/profile/ProfileShowcaseView.qml [hunk 1]
```diff
@@ -100,8 +94,8 @@ Control {
                     title: model.name
                     statusListItemTitle.font.pixelSize: 17
                     statusListItemTitle.font.bold: true
-                    subTitle: model.description
-                    tertiaryTitle: qsTr("%n member(s)", "", model.members.count)
+                    //subTitle: model.description // TODO: no data in showcase model
+                    //tertiaryTitle: qsTr("%n member(s)", "", model.members.count) // TODO: no data in showcase model
```
Comment (jrainville) [inline ui/imports/shared/views/profile/ProfileShowcaseView.qml:98]: What do you mean? We don,t support those in the model yet?

File: ui/imports/shared/views/profile/ProfileShowcaseView.qml [hunk 2]
```diff
@@ -175,7 +169,7 @@ Control {
                     components: [
                         StatusIcon {
                             anchors.verticalCenter: parent.verticalCenter
-                            visible: model.walletType === Constants.watchWalletType
+                            //visible: model.walletType === Constants.watchWalletType
```
Comment (jrainville) [inline ui/imports/shared/views/profile/ProfileShowcaseView.qml:172]: Is this a leftover?

File: src/app/modules/main/profile_section/profile/module.nim [hunk 3]
```diff
@@ -107,34 +109,77 @@ method storeProfileShowcasePreferences(self: Module,
...
   var profileCommunityItems: seq[ProfileShowcaseCommunityItem] = @[]
   var profileAccountItems: seq[ProfileShowcaseAccountItem] = @[]
   var profileCollectibleItems: seq[ProfileShowcaseCollectibleItem] = @[]
   var profileAssetItems: seq[ProfileShowcaseAssetItem] = @[]
 
-  for communityEntry in communityEntries:
-    let community = self.controller.getCommunityById(communityEntry.id)
-    profileCommunityItems.add(initProfileShowcaseCommunityItem(community, communityEntry))
+  for communityEntry in contact.profileShowcase.communities:
+    # TODO: what if we don't know such community?
```
[the remark this answers is not in the document]
Comment (MishkaRogachev) [reply src/app/modules/main/profile_section/profile/module.nim:131]: I'm going to just hide loading items, but i need a few changes from https://github.com/status-im/status-desktop/pull/12613

File: ui/imports/shared/views/profile/ProfileShowcaseView.qml (hunk 2 above)
Comment (MishkaRogachev) [reply ui/imports/shared/views/profile/ProfileShowcaseView.qml:172]: leftover, we already check this when filling the showcase model

File: src/app/modules/main/profile_section/profile/module.nim (hunk 3 above)
[the remark this answers is not in the document]
Comment (MishkaRogachev) [reply src/app/modules/main/profile_section/profile/module.nim]: fixed

[the remark this answers is not in the document]
Comment (jrainville) [reply src/app/modules/main/profile_section/profile/module.nim:131]: I don't see the changes here

File: ui/imports/shared/views/profile/ProfileShowcaseView.qml (hunk 1 above)
Comment (jrainville) [reply ui/imports/shared/views/profile/ProfileShowcaseView.qml:98]: This one also didn't chnage

**Review by jrainville — COMMENTED** [review]
I think you forgot to push

**Comment by MishkaRogachev** [comment]
> `3cee0b1`

Gosh! I force-pushed from a wrong working copy while testing. Hopefully, [commit](https://github.com/status-im/status-desktop/commit/3cee0b1db3ccf98c0580a3114f5db71eeec8fd83) did not get lost.

Now should be fine

**Review by caybro — APPROVED** [review]
QML changes LGTM

**Review by jrainville — APPROVED** [review]
Looks good

**Comment by MishkaRogachev** [comment]
https://github.com/status-im/status-desktop/assets/2522130/1b284d14-58cd-4c8c-b783-d3326ed20dec

Proof of working with a fresh, perviously unknown community

**Comment by MishkaRogachev** [comment]
Also storybook...

https://github.com/status-im/status-desktop/assets/2522130/1610dd04-5d5e-4ce7-8c12-bba055b6c068

**Comment by MishkaRogachev** [comment]
Screenshot 2023-11-10 at 00 23 50
Screenshot 2023-11-10 at 00 23 45
Screenshot 2023-11-10 at 00 23 41
Screenshot 2023-11-10 at 00 23 37

Storybook test

**Comment by jrainville** [comment]
Force merging as the `tests/ui` are failing on start for no reason, seems like a host issue.
Mishka proved that they work locally.

**Pull request merged**